### ISSUE TITLE
Feature Request:  Play all / Pause all

### ISSUE THREAD

**Issue by Bub-la-bub** [iina/iina#2447]
Feature request to add Play All / Pause All functionality similar to Quicktime 7's "Play All Movies (⌘↩︎)" feature.  This allows you to have multiple clips open at once and play or pause them all simultaneously with the command-return key combo.  Very handy when comparing clips.

**Comment by flocked** [comment]
+1 for this feature. I have sometimes several videos at the same time open. Being able to quickly pause them all at the same time, would be great.

**Comment by drendon** [comment]
I totally agree. I came today to create a feature request for this, but found this duplicate.

This is the one feature of QuickTime Player 7 that I use regularly and that IINA did not replace. I use it to compare different compression levels or small changes in otherwise identical clips. It is one of the reasons why I won't upgrade to Catalina any time soon (QT Player 7 is 32-bit).

**Comment by sxflynn** [comment]
I think this would be a great feature that I can use to compare different clips at different levels of compression to find the most optimal one.
